Gravity-tube/src/Components/Video-Listing/VideoListing.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import VideoCard from "../VideoCard/VideoCard";
+import "./VideoListing.css";
+
+function VideoListing() {
+  return (
+    <div className="video-box">
+      <h2>Recommended</h2>
+      <div className="videos-container">
+        <VideoCard 
```

Comment:
I think you have to take the data from the videos endpoint right and then loop over it?

Reply:
yes its a static data